frontend/src/store/modules/visitors/actions.js

```diff
@@ -30,6 +31,29 @@ export default {
             context.commit(RESET_BUTTON_FETCHING, data.button);
         }
     },
+    [GET_LINE_CHART_DATA]: (context, data) => {
+        if (!data.value) {
+            return;
+        }
+
+        context.commit(SET_LINE_CHART_DATA_FETCHING);
+        context.commit(GET_SELECTED_PERIOD)
```

Comment:
commit doesn't return promise, also mutation shouldn't return data, use state from context object

in general all chain, that implemented below, won't work